docs/src/layouts/DocLayout.vue

```diff
@@ -76,7 +76,7 @@ export default {
     const $q = useQuasar()
     const $route = useRoute()
     const headerScrollData = ref()
-    const HEADER_SCROLL_OFFSET = 100 // number of pixels of scroll before firing a possible header scroll
+    const HEADER_SCROLL_OFFSET = 64 // number of pixels of scroll before firing a possible header scroll
```

Comment:
This should be shared between the layout and the main layout header, we need to put it into a secondary file imported by both components